packages/commercetools/theme/pages/CheckoutV2/Payment.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <SfHeading
+      :level="3"
+      :title="$t('Billing')"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <BillingForm
```

Comment:
why we need to pass it instead of just invoking composable there? If you want to follow smart/dumb pattern then again - in other components that are not pages you're also using composables (+ we don't have this pattern in our app anyway)

Reply:
Payment.vue is responsible for saving&loading and BillingForm is only responsible form-related stuff. I wanted to separate these parts. However, over iterations of CRs both Payment.vue & Shipping.vue getting smaller and smaller. Shipping.vue in newest PR for shipping looks like that:
```vue
<template>
  <div>
    <ShippingForm
      :handleShippingAddressSubmit="submitShippingAddress"
    />
  </div>
</template>
```

Look at it, we could get rid of both ShippingForm & BillingForm. ShippingForm should become Shipping.vue and BillingForm should become Payment.vue. Wdyt?